react/src/components/manager.jsx

```diff
@@ -124,7 +124,13 @@ class CreditAccount extends React.Component {
     }
 
     handleChange(event) {
-        this.setState({ [event.target.name]: event.target.value });
+        if (event.target.name == "amount") {
+            const cleanedValue = event.target.value.replaceAll("_", "0") // Remove the remaining maskChar
+            const euroAndCents = cleanedValue.split(/,|\s/); // Use regex to catch mutiple delimiters
+            this.setState({ "amount": euroAndCents[0] + euroAndCents[1]});       
+        } else {
+            this.setState({ [event.target.name]: event.target.value });
+        }
```

Comment:
This code is not working, in fact when you just enter 1 it input automatically 100€.
This seems to be a problem of the way to input data. Here, it fill from left to right instead of right to left as in `create_conso`.

Reply:
It seems like react-input-mask doesn't support reverse input, I'll try to implement it with react-maskedinput following [this tuto](https://www.nicknish.co/blog/react-currency-input)

Reply:
I change the component for a traduction in React of [this typescript code](https://github.com/larkintuckerllc/react-currency-input/blob/master/src/App.tsx). The behavior are same now.